lenovo/thinkpad/e14/intel/default.nix

```diff
@@ -3,8 +3,10 @@
 {
   imports = [
     ../.
-    ../../../../common/cpu/intel
+    ../../../../common/cpu/intel/comet-lake
   ];
 
+  hardware.intelgpu.vaapiDriver = "intel-media-driver";
```

Comment:
Do we not set this in the intel module? Maybe we should.

Reply:
As far as I understood it, it by default pulls in intel-media-driver and intel-vaapi-driver but I don't think we need the later.

Reply:
We have several intel generation profiles configuring `intel-media-driver`. If this is the right one for intel comet lake, we should do the same.